components/support/ktx/src/test/java/mozilla/components/support/ktx/android/os/StrictModeTest.kt

```diff
@@ -0,0 +1,31 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.support.ktx.android.os
+
+import android.os.StrictMode
+import org.junit.Assert.assertEquals
+import org.junit.Assert.assertNotEquals
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.robolectric.RobolectricTestRunner
+
+@RunWith(RobolectricTestRunner::class)
+class StrictModeTest {
+    @Test
+    fun `strict mode policy should be restored`() {
```

Comment:
We should also ensure the thread policy is restored if the function block throws an error. :)